pkg/subctl/operator/submarinerop/scc/ensure.go

```diff
@@ -23,5 +23,11 @@ import (
 )
 
 func Ensure(restConfig *rest.Config, namespace string) (bool, error) {
-	return scc.UpateSCC(restConfig, namespace, serviceaccount.OperatorServiceAccount)
+	operatorUserAdded, err := scc.UpateSCC(restConfig, namespace, serviceaccount.OperatorServiceAccount)
```

Comment:
unrelated to your patch, we have a typo in UpateSCC -> UpdateSCC

Reply:
Fixed the typo